### DISCUSSION TITLE
Does fzf support printing only the first match in --filter mode?

### DISCUSSION THREAD

**Discussion by haphamdev** [junegunn/fzf#2419] [Q&A] [answered]
For example:

```
$ cat > file.txt
Lorem ipsum dolor
sit amet
consectetur adipiscing elit
sed do eiusmod
tempor incididunt


$ cat file.txt | fzf --filter so
sed do eiusmod
Lorem ipsum dolor
```

Is there any way to make `fzf` print only the first match on `--filter` mode? 
In this case, I would like to print `sed do eiusmod`, not the line `Lorem ipsum dolor`.

**Comment by haphamdev** [comment]
I can achieve the same result with  `cat file.txt | fzf --filter so | head -n1` but I am wondering whether fzf has any feature like that.

**Comment by junegunn** [comment, marked as answer]
fzf follows the [Unix philosophy](https://en.wikipedia.org/wiki/Unix_philosophy) and chooses not to implement features that can be done by other programs; So using `head` is the recommended way to do it.
